NonEmergencyAmbulanceTransportation/R4/resources/Questionnaire-R4-NonEmergencyAmbulanceTransportationProgressNote.json

```diff
@@ -67,11 +71,11 @@
               "url": "http://hl7.org/fhir/StructureDefinition/cqf-expression",
               "valueExpression": {
                 "language": "text/cql",
-                "expression": "\"NonEmergencyAmbulanceTransportationPrepopulation\".RelevantDiagnoses"
+                "expression": "\"NonEmergencyAmbulanceTransportationPrepopulation\".AllDiagnoses"
               }
             }
           ],
-          "text": "Patient Diagnoses related to need for a non-emergency ambulance transportation",
+          "text": "Principal Diagnoses",
```

Comment:
I would suggest using "Relevant Diagnoses" to be consistent with other forms although internally we are using "AllDiagnoses" for the expression.